public/components/content-list-item/_content-list-item.scss

```diff
@@ -541,3 +544,10 @@
         display: inline-block;
     }
 }
+
+.planned-print-location {
+    color: grey;
```

Comment:
Could we re-use something from the [palette](https://github.com/guardian/workflow-frontend/blob/master/public/layouts/global/_colours.scss)?

Reply:
Done.